src/Nethermind/Nethermind.Api/IInitConfig.cs

```diff
@@ -85,6 +85,9 @@ public interface IInitConfig : IConfig
 
     [ConfigItem(Description = "[TECHNICAL] Exit when block number is reached. Useful for scripting and testing.", DefaultValue = "null", HiddenFromDocs = true)]
     long? ExitOnBlockNumber { get; set; }
+
+    [ConfigItem(Description = "[TECHNICAL] Produce block for every possible slot.", DefaultValue = "false", HiddenFromDocs = true)]
```

Comment:
Think about a bit better description, simulate block production? for dev purposes ?